Update/some_004.txt

```diff
@@ -3657,13 +3657,13 @@ void main()
 //rぶるぶると小刻みに震え続ける千紗登ちゃん。震えを押し殺そうと、更にギュッと両手を握り締める。
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 	OutputLine(NULL, "ぶるぶると小刻みに震え続ける千紗登ちゃん。震えを押し殺そうと、更にギュッと両手を握り締める。",
-			NULL, "Little by little, Chisato-chan continued trembling. She braced herself with both hands to ease the shaking.", Line_Normal);
+			NULL, "Chisato-chan's shoulders gradually continued to tremble intensely. She braced herself with both hands to ease the shaking.", Line_Normal);
```

Comment:
```suggestion
			NULL, "Chisato-chan's shoulders continued to tremble more and more. She braced herself with both hands to ease the shaking.", Line_Normal);
```
Something can't continue gradually. (As much as I'd like it to sometimes...) Alternate phrasing suggested.